tools/linter/clang_tidy/__main__.py

```diff
@@ -93,7 +94,13 @@ def clang_search_dirs() -> List[str]:
         "-torch/csrc/deploy/test_deploy_python_ext.cpp",
     ],
     "paths": ["torch/csrc/"],
-    "include-dir": ["/usr/lib/llvm-11/include/openmp"] + clang_search_dirs(),
+    "include-dir": [
+        "/usr/lib/llvm-11/include/openmp",
+        # We want the string '/usr/local/include/python<version number>'
+        # https://stackoverflow.com/a/35071359
```

Comment:
please put it in a function call it something like:
`def get_python_include_dir() -> str: return gp()['include'] `

Reply:
Done